.vib/discourse/cypress/cypress/integration/discourse_spec.js

```diff
@@ -0,0 +1,64 @@
+/// <reference types="cypress" />
+import { random } from './utils';
+
+it('allows to log in and out', () => {
+  cy.login();
```

Comment:
After this line, I'd also add a test that searches for error messages in the login.
That way if login ever fails, we can know from the Cypress report the failure is in the login test.
Something like this
`cy.get('#error-message').should('not.exist)`